.travis.yml

```
@@ -5,63 +5,86 @@ addons:
   apt:
     packages:
     - socat
-    - enchant
-    - aspell
-    - aspell-en
 
 env:
   global:
-  - REDIS_TAGS="2.6.17 2.8.22 3.0.7 3.2.8 4.0.11 5.0.1" INSTALL_DIR=$HOME/redis
+    REDIS_TAGS: >-
+      2.6.17 2.8.22 3.0.7 3.2.8 4.0.11 5.0.1
```

Oh, I've made a mistake, it should be:
```suggestion
      '2.6.17 2.8.22 3.0.7 3.2.8 4.0.11 5.0.1'
```
(because of spaces)